webapp/src/views/DeleteAccount.vue

```diff
@@ -101,10 +93,6 @@
       /* password field */
       password: '',
       password_errors: [],
-
-      /* email field */
-      new_email: '',
-      email_errors: [],
     }),
     mounted() {
       if ('email' in this.$route.params && this.$route.params.email !== undefined) {
```

Comment:
This `if` branch can also go away. Perhaps move to af53e1a23d74ff1996c514df15140353fb370f31 (`fix(webapp): removes dead code`), along with removal of `new_email` and `email_errors`?